src/main/java/org/killbill/billing/plugin/adyen/client/payment/builder/checkout/CheckoutPaymentsBuilder.java

```diff
@@ -151,16 +155,28 @@ private void setShopperData() {
         if (userData.getGender() != null) {
             name.setGender(Name.GenderEnum.valueOf(userData.getGender().toUpperCase()));
         }
+
         if (userData.getFirstName() != null ||
             userData.getInfix() != null ||
             userData.getLastName() != null ||
             userData.getGender() != null) {
             request.setShopperName(name);
         }
 
-        request.setShopperLocale(userData.getShopperLocale().toString());
+        // Split the locale text with only 2 chars
+        // e.g. Use 'de' if given locale is de_DE
+        if(userData.getShopperLocale() != null) {
```

Comment:
will there be any impact on other adyen call?

Reply:
No. This is only for new Checkout API.